### ISSUE TITLE
iOS client

### ISSUE THREAD

**Issue by jitterfrei** [martinstoeckli/SilentNotes#208]
**Describe the solution you'd like**
I'm happily using SilentNotes for years now to sync my notes between Windows and Android. Now I'm in the process of switching to an iPhone and realized there is no iOS version of SilentNotes (yet). From what I understood from other Issues, this is on the roadmap? However, I couldn't find any further information on it. Since for me this program is the best note taking app out there, I really don't want to move to any other app. So, is there any ongoing development for an iOS version? If so, I would be happy to support you and join development.

**Comment by martinstoeckli** [comment]
Technically it would be possible to write an iOS version, most of the code could be reused. Unfortunately the hurdles are quite high, Apple charges an annual developer fee of $100 to use their store (vs Android $25 and Microsoft $20 one-time fee). Then one is required to own a Mac to be able to upload an app to the store, and last but not least it I would have to buy an iPhone to do serious testing.

I already played around with it a bit, but I don't think there will be an iOS version any time soon, sorry. I got the impression that Apple is interested more in commercial than freeware products, which has the advantage that less poor quality software finds its way into their store.

**Comment by jitterfrei** [comment]
I would also take over development of the iOS version completely (I own an iPhone and have access to a Mac), but the annual fee is indeed a huge hurdle. However, [from what I read recently](https://www.spiegel.de/netzwelt/apps/apple-bereitet-sich-offenbar-auf-alternative-app-stores-in-ios-17-vor-a-eddc5d19-8a6e-4145-aa9d-a76eb331ab7f) Apple plans to open their system up for 3rd party app stores later this year on pressure from the EU, so hopefully at least that issue will be gone soon.

**Comment by martinstoeckli** [comment]
It would be great to have an iOS version and if you are experienced in software development I would welcome help!
Apple allowing for 3rd party stores would be a big step, btw (Swiss-)German is my native language so the link would have worked out quite well 😀

**Comment by jitterfrei** [comment]
Yep, I thought that you would probably understand it, but with other possible readers in mind I thought English might be better. Anyway, since the English article is behind a paywall, [here is the German link as well](https://www.spiegel.de/netzwelt/apps/apple-bereitet-sich-offenbar-auf-alternative-app-stores-in-ios-17-vor-a-eddc5d19-8a6e-4145-aa9d-a76eb331ab7f).

I have about 15 years of experience in SW development using various languages including some Apps for Android in Java and Kotlin. I havn't used C#/Typescript so far, but I don't think it will be an issue and I was curious about developing something for iOS anyway, so why not for my favorite note taking app 😉

I will look into the code a bit, but maybe at some point it would be good to have a short discussion on the codebase and your experience with an iOS version so far.
